src/controller/python/chip-device-ctrl.py

```diff
@@ -762,6 +762,40 @@ def resolve(fabric_id: int, node_id: int) -> Dict[str, Any]:
     except Exception as e:
         return __get_response_dict(status = StatusCodeEnum.FAILED, error = str(e))
 
+def zcl_command(cluster: str,command: str, node_id: int, endpoint_id: Optional[int] = 1, group_id: Optional[int] = 0, args: Optional[dict] = {}) -> Dict[str, Any] :
+    try:
+        __check_supported_os()
+
+        all_commands = device_manager.devCtrl.ZCLCommandList()
+        command_arg = all_commands.get(cluster).get(command, None)
+        if command_arg is None:
+            raise exceptions.UnknownCommand(cluster, command)
+        commandArgs = __format_zcl_arguments_from_dict(args, command_arg)
+
+        err, res = device_manager.devCtrl.ZCLSend(cluster, command, node_id, endpoint_id, group_id, commandArgs, blocking=True)
+        if err != 0:
```

Comment:
can simply be `if err:`

Reply:
This is value check, if err gets 0 as a value the API call passed for any value other than 0 the API call failed

Reply:
right but `if 0` and `if None` evaluate to false and they will not enter the if statement, which is the desired behavior right?

Reply:
Ack